StatusChanger.js

```diff
@@ -17,30 +17,36 @@ $(function() {
   var setMessage = function(stat) {
     var message = stat;
     switch (message) {
-      case "sleeping":
+      case "zzz":
         message = "asleep";
         break;
-      case "BOW(A)":
+      case "BOW-A":
         message = "very busy dealing with something on-wiki";
```

Comment:
This should be as below
```suggestion
        message = "very dealing with something on-wiki";
```